src/Components/ExternalResult/FacilitiesSelectDialogue.tsx

```diff
@@ -30,7 +30,7 @@ const FacilitiesSelectDialog = (props: Props) => {
         showAll={false}
         multiple={false}
       />
-      <div className="mt-4 flex justify-between">
+      <div className="mt-4 flex flex-col">
```

Comment:
This PR contains changes from your other PR. Ensure you've branched from the `develop` branch instead of the other PR's branch.

Reply:
Okay will Do